source/administration/config-settings.rst

```diff
@@ -4037,6 +4037,102 @@ Changes to this setting require a server restart before taking effect.
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: When ``true`` file output is enabled.
+**False**: File output is disabled.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileEnabled": []`` with options ``true`` and ``false``.                               |
++-------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File name
+^^^^^^^^^^^^^^
+This is the path to the output file location.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileName": []`` with string array input consisting of a user-defined name.                     |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max size MB
+^^^^^^^^^^^^^^
+This is the maximum size (measured in megabytes) that the file can grow before triggering rotation. The default setting is 100.
+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileMaxSizeMB": []`` with numerical input.                                                    |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max age days
+^^^^^^^^^^^^^^
+This is the maximum age in days a file can reach before triggering rotation. There is no default limit, and no limit to the value used.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileMaxAgeDays": []`` with numerical input.                                                    |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max backups
+^^^^^^^^^^^^^^
+This is the maximum number of rotated files kept; oldest is deleted first.
```

Comment:
@wiggin77, is there a limit on the value that can be entered? Is there a default value?

Reply:
Ah yes, I left out some info.  

FileMaxSizeMB defaults to 100
FileMaxAgeDays defaults to 0;  0 means no limit on age
FileMaxBackups defaults to 0;  0 means no limit on number of backups

Reply:
None of those have any meaningful maximum value.

Reply:
Thanks!